src/hooks/useGetUserByPosition.ts

```diff
@@ -12,43 +11,60 @@ export interface UseGetUserByPositionProps {
   venueId?: string;
 }
 
-export type GetUserByPostion = (
+export type GetUserByPosition = (
   gridPosition: GridPosition
-) => WithId<User> | undefined;
+) => WithId<GridSeatedUser> | undefined;
 
-export const useGetUserByPosition: (
+const getPositionHash = ({ row, column }: GridPosition): string => {
+  return `${row}|${column}`;
+};
+
+export const useGetUserByPositionOld: (
   props: UseGetUserByPositionProps
-) => GetUserByPostion = ({ positionedUsers, venueId }) => {
-  const seatedUsersByHash: Map<string, WithId<User>> = useMemo(
+) => GetUserByPosition = ({ positionedUsers, venueId }) => {
+  const gridSeatedUsers = venueId
+    ? positionedUsers.map((user) => {
+        const { row, column } = user.data?.[venueId] ?? {};
+
+        return {
+          ...user,
+          position: {
+            row,
+            column,
+          },
+        };
+      })
+    : [];
```

Comment:
Does he still have this data?

Reply:
I've renamed the old hook to `useGetUserByPositionOld` as it's still being used inside `Map.tsx`
But probably will get rid of that too when we finish with other venues.